DataFormats/METReco/src/classes_def.xml

```diff
@@ -133,7 +133,7 @@
    <version ClassVersion="12" checksum="4280559065"/>
    <version ClassVersion="13" checksum="382065414"/>
    <version ClassVersion="14" checksum="651605939"/>
-   <version ClassVersion="15" checksum="2415916091"/>
+   <version ClassVersion="15" checksum="1886001321"/>
```

Comment:
Please, put back the previous class version, and move to a new one ("16") because of the added class member